### ISSUE TITLE
SDL formatting for resolver argument comments regressed in 6.0.5

### ISSUE THREAD

**Issue by djc** [async-graphql/async-graphql#1363]
## Changed SDL output

```diff
--- a/typescript/packages/instant-shared/src/graphql/schema.graphql
+++ b/typescript/packages/instant-shared/src/graphql/schema.graphql
@@ -179,9 +179,9 @@ enum AvailablePhoneNumbersErrorKind {
 
 input AvailablePhoneNumbersInput {
        site: ID!
-       """
-       See `supportedPhoneNumberCountries`
-       """
+               """
+               See `supportedPhoneNumberCountries`
+               """
        countryCode: String!
 }
 
```

## Specifications

- Version: 6.0.5
- Platform: macOS
- Subsystem: ?

**Comment by sunli829** [comment]
fixed in `6.0.6` 🙂

**Comment by djc** [comment]
Thanks!
